test/infra/repository/world_state_repository_with_leveldb_test.cpp

```diff
@@ -0,0 +1,60 @@
+//
+// Created by SonokoMizuki on 2016/10/23.
+//
+
+
+#include "../../../core/repository/world_state_repository.hpp"
+
+#include <leveldb/db.h>
+
+#include <gtest/gtest.h>
+#include <iostream>
+#include <string>
+
+const std::string key = "name";
+std::string value = "mizuki";
+
+
+TEST(World_sate_repository_with_leveldb, AddAndfind){
+
+    repository::world_state_repository::add(key, value);
+
+    std::string res;
+    res = repository::world_state_repository::find(key);
+    ASSERT_STREQ(res.c_str(), value.c_str());
+}
+
+
+TEST(World_sate_repository_with_leveldb, NotFind){
+    std::string res;
+    res = repository::world_state_repository::find(key+"++");
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Variable 'res' is assigned a value that is never used.](https://www.codacy.com/app/soramitsu/iroha/file/4256572247/issues/source?bid=3712501&fileBranchId=3831158#l30)